service/src/main/java/io/camunda/service/TenantServices.java

```diff
@@ -79,5 +81,28 @@ public TenantEntity getByKey(final Long tenantKey) {
     return tenantEntity;
   }
 
+  public CompletableFuture<?> addMember(
+      final Long tenantKey, final EntityType entityType, final long entityKey) {
+    return modifyEntity(tenantKey, entityType, entityKey, true);
+  }
+
+  public CompletableFuture<?> removeMember(
+      final Long tenantKey, final EntityType entityType, final long entityKey) {
+    return modifyEntity(tenantKey, entityType, entityKey, false);
+  }
+
+  private CompletableFuture<?> modifyEntity(
+      final long parentKey,
+      final EntityType entityType,
+      final long entityKey,
+      final boolean isAdd) {
```

Comment:
Boolean flags make this a bit harder to read (e.g. when I look at lines 86 and 91 I don't know what the boolean means). Since this is so small, can we inline modifyEntity() instead?